src/native/corehost/fxr/sdk_resolver.cpp

```diff
@@ -40,11 +40,16 @@ sdk_resolver::sdk_resolver(fx_ver_t version, sdk_roll_forward_policy roll_forwar
 {
 }
 
-pal::string_t const& sdk_resolver::global_file_path() const
+const pal::string_t& sdk_resolver::global_file_path() const
 {
     return global_file;
 }
 
+const fx_ver_t& sdk_resolver::requested_version() const
+{
+    return version;
```

Comment:
Nit: could you please rename `version` to `requested_version`? I was looking through the code it if was rather confusing. Also we have some really unfortunate local conflicts, for example: https://github.com/dotnet/runtime/blob/de73d8205047ba946299dc370adbb92f777ab560/src/native/corehost/fxr/sdk_resolver.cpp#L493 this function uses the field at first and then it has a foreach which declares `version` as a local variable.

Reply:
That is unfortunate indeed 😕